openedx/core/djangoapps/edx_global_analytics/tests/test_platform_coordinates.py

```diff
@@ -0,0 +1,102 @@
+"""
+Tests for edx_global_analytics application tasks and helper functions.
+"""
+
+import unittest
+
+import requests
+from mock import patch
+
+from ..utils import (
+    get_coordinates_by_platform_city_name,
+    get_coordinates_by_ip
+)
+
+
+@patch('openedx.core.djangoapps.edx_global_analytics.utils.requests.get')
+class TestPlatformCoordinates(unittest.TestCase):
```

Comment:
These are some pretty good unit tests! 👍 
Test names are probably a bit too long. But that's fine.